docs/cryptography/caesar-cipher.mdx

```diff
@@ -0,0 +1,229 @@
+    for character in range(len(text)):
+
+        # Saving current character in current_character
+        current_character = text[character]
+
+        # If current characater is space adding it directly to message
+        if current_character == ' ':
+            message += current_character
+            continue
+
+        # If current characater is an uppercase letter adding it after shifting with key value to message
+        if current_character.isupper():
+            message += chr((ord(current_character) - shift - 65) % 26 + 65)
+            continue
+
+        # If current characater is an lowercase letter adding it after shifting with key value to message
+        if current_character.islower():
+            message += chr((ord(current_character) - shift - 97) % 26 + 97)
+            continue
+
+    # Returning the message
+    return message
+
+
+# Encrypt Caesar Cipher Messages
+
+# Function to encrypt a message
+def encrypt(text, shift):
+    message = ""
+
+    # Looping through all the characters present in text
+    for character in range(len(text)):
+
+        # Saving current character in current_character
+        current_character = text[character]
+
+        # If current characater is space adding it directly to message
+        if current_character == ' ':
+            message += current_character
+            continue
+
+        # If current characater is an uppercase letter adding it after shifting with key value to message
+        if current_character.isupper():
+            message += chr((ord(current_character) + shift - 65) % 26 + 65)
+            continue
+
+        # If current characater is an lowercase letter adding it after shifting with key value to message
+        if current_character.islower():
+            message += chr((ord(current_character) + shift - 97) % 26 + 97)
+            continue
+
+    # Returning message
+    return message
+
+
+# Driver main function
+if __name__ == '__main__':
+    choice = input("Choose 1 for ENCRYPT and 2 for DECRYPT : ")
+    if int(choice) == 1:
+        print("Enter message to encrypt")
+        text = input()
+        print("Enter key value")
+        shift = int(input())
+        encrypted_message = encrypt(text, shift)
+        print("Encrypted message : " + encrypted_message)
+
+    elif int(choice) == 2:
+        print("Enter message to decrypt")
+        text = input()
+        print("Enter key value")
+        shift = int(input())
+        decrypted_message = decrypt(text, shift)
+        print("Decrypted message : " + decrypted_message)
+
+    else:
+        print("Wrong Choice")
+```
+
+</TabItem>
+</Tabs>
+
+## Sample Input and Output
+
+### Input:
+
+```
+Text : TESSERACTCODING
+Shift: 3
+```
+
+### Output:
+
+```
+Cipher: WHVVHUDFWFRGLQJ
+```
+
+### Input:
+
+```
+Line 1: 1
+Line 2: "Hello"
+Line 3: 2
+```
+
+### Output:
+
+```
+Encrypted message : Jgnnq
+```
+
+## Complexity Analysis
+
+    - Time complexity - O(n)
+    - Space complexity - O(n)
```

Comment:
No need to keep them in a block. Keep them as a normal text and use backticks around the values